.devcontainer/Dockerfile

```diff
@@ -1,20 +1,6 @@
-# See here for image contents: https://github.com/devcontainers/images/tree/main/src/go
-
-# [Choice] Go version (use -bullseye variants on local arm64/Apple Silicon): 1, 1.16, 1.17, 1-bullseye, 1.16-bullseye, 1.17-bullseye, 1-buster, 1.16-buster, 1.17-buster
-ARG VARIANT="1.23-bullseye"
-FROM mcr.microsoft.com/vscode/devcontainers/go:dev-${VARIANT}
+FROM mcr.microsoft.com/oss/go/microsoft/golang:1.23.6-bookworm@sha256:2d8c09c93157b49f2f8f0523d907620d022e6d354e670b60a83125067f0c8960
 
 # [Optional] Uncomment this section to install additional OS packages.
 # graphviz for pprof
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install --no-install-recommends graphviz
-
-RUN curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v2.13.0/skaffold-linux-amd64 && \
```

Comment:
Dont need to install skaffold anymore?

Reply:
Still need it, but installed as devcontainer feature instead (bumped to latest as side-effect)